### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by pbkabali**
## What this PR does:
- Add component structure of the app
- Implement the Display component
- Implement the ButtonPanel component
- Implement the button component

[Live link](https://shielded-everglades-20664.herokuapp.com/)

### CODE REVIEW

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  const { result } = props;
+  return <div>{result}</div>;
+};
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+
+Display.defaultProps = {
+  result: 0,
```
Comment (whiz25) [inline src/components/Display.js]: - The value of `result` should be a string
```javascript
result: '0'
```

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required :pray: 
Nice work @pbkabali you did well in this PR. The codebase is well structured. All components are rightly placed in the `components` directory.
Make the following changes so we can merge this PR.
- [x] add description to your PR. Tell us what the PR is about, the features that the PR covers.
- [x] the result of the calculation should be a string. check my comment for suggestion


### Martin Afani

**Comment by pbkabali** [comment]
Thanks a lot @whiz25 for the review. I have made improvements and made a resubmission.

**Review by christianotieno — APPROVED** [review]
## Milestone Approved 🍾 
Hello @pbkabali 👋 
The changes are now reflecting 👏 
I wish you all the best in your next tasks 🥇 

### Happy coding 💻 
#### Christian Otieno (CR)

**Comment by pbkabali** [comment]
Thank you very much @christianotieno

**Pull request merged**